libs/javascript/package.json

```
@@ -0,0 +1,16 @@
+{
+  "name": "fetch-repo-files",
+  "version": "1.0.0",
+  "description": "Script to fetch metadata files from ",
+  "main": "metadatafetcher.js",
+  "scripts": {
+    "fetch:metadata": "node metadatafetcher.js"
+  },
+  "author": "Christian Lechner",
+  "license": "Apache2",
+  "dependencies": {
+    "@octokit/auth-token": "3.0.0",
+    "@octokit/request": "6.2.0",
+    "node-fetch": "2.6.7"
```

Pinning and reproducible builds are covered by package-lock + "npm ci"; here we can use an open range:

```suggestion
    "@octokit/auth-token": "^3.0.0",
    "@octokit/request": "^6.2.0",
    "node-fetch": "^2.6.7"
```

---

Keeping the hard pinned deps in package.json